apps/labs/posts/scipy-array-api.md

```diff
@@ -0,0 +1,441 @@
+---
+title: 'Array API Standard in SciPy'
+published: September 21, 2023
+authors: [lucas-colley]
+description: "How can SciPy use the Array API Standard to achieve array library interoperability?"
+category: [Array API]
+featuredImage:
+  src: /posts/scipy-array-api/scipy-array-api-illustration.png
+  alt: 'An illustration of three cartoon characters - CuPy Cube, SciPy Snake and PyTorch Flame - together (happily!) under a tree with green leaves bearing the Consortium for Python Data API Standards logo ("DataAPIs") on a brown background. CuPy Cube and PyTorch Flame are made up of the CuPy and PyTorch logos respectively, with smiley faces and stick legs. SciPy Snake is a green snake with a red tongue, wearing sunglasses with SciPy logos for lenses.'
+hero:
+  imageSrc: /posts/scipy-array-api/scipy-array-api-hero.png
+  imageAlt: 'An illustration of three cartoon characters - CuPy Cube, SciPy Snake and PyTorch Flame - together (happily!) under a tree with green leaves bearing the Consortium for Python Data API Standards logo ("DataAPIs") on a brown background. CuPy Cube and PyTorch Flame are made up of the CuPy and PyTorch logos respectively, with smiley faces and stick legs. SciPy Snake is a green snake with a red tongue, wearing sunglasses with SciPy logos for lenses. Some graphical elements on a white background surround the illustration.'
+---
+
+In this blog post I discuss how SciPy is harnessing the power of array-agnostic code to become
+more interoperable, and why that matters.
+
+---
+
+Hi 👋 I'm [Lucas Colley](https://github.com/lucascolley), a Computer Science & Philosophy student
+from the UK.
+This summer, I have had the amazing opportunity of working under the mentorship of
+[Ralf Gommers](https://github.com/rgommers), [Pamphile Roy](https://github.com/tupui) and
+[Irwin Zaid](https://github.com/izaid) in my Open Source internship at Quansight Labs.
+My project was to make progress towards SciPy's goal of adopting support for the array API standard,
+which will allow users to use SciPy functions with CuPy arrays, PyTorch Tensors and potentially
+many other array/tensor libraries in the future.
```

Comment:
Nice introduction! I think it's clear, very concisely describes the goal, and references the previous work appropriately.